core/commands/commands_test.go

```diff
@@ -233,6 +233,13 @@ func TestCommands(t *testing.T) {
 		"/storage/challenge",
 		"/storage/challenge/request",
 		"/storage/challenge/response",
+		"/storage/stat",
+		"/storage/stat/info",
+		"/storage/stat/sync",
```

Comment:
To follow this convention, the command should be `stats`: https://docs-beta.ipfs.io/reference/cli/#ipfs-stats

Reply:
got it, will update code and design doc